app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/ItemAdapter.kt

```diff
@@ -30,7 +30,7 @@ class ItemAdapter(var context: Context, var view_layout: Int, var arrayCaseList:
         var deaths: TextView = view.findViewById<TextView>(R.id.list_deaths)
 
         var items: Cases = arrayCaseList.get(position)
-        title.text = items.country_name[0]
```

Comment:
Changed to better fit Kotlin naming conventions https://kotlinlang.org/docs/reference/coding-conventions.html